src/CircleGraph/CircleGraphCtrl.ts

```diff
@@ -145,27 +145,29 @@ export class CircleGraphCtrl {
   private registerEventHandlers() {
     // Handle messages from the webview
     this._webview.onDidReceiveMessage(message => {
+      switch (message.command) {
+        case MessageDefs.alert:
+          Balloon.error(message.text);
+          return;
+        case MessageDefs.request:
+          this.handleRequest(message.url, message.encoding);
+          return;
+        case MessageDefs.loadmodel:
+          this.handleLoadModel(parseInt(message.offset));  // to number
+          return;
+        case MessageDefs.finishload:
+          this.handleFinishLoad();
+          return;
```

Comment:
Although `handleReceiveMessage` has no `default` handling, maybe it would be better to have? :)

```suggestion
          return;
        default:
          Balloon.error('No command found');
```

---

And this is just for curious, why only some of commands are relocated?